--- uriparser-0.9.3/CMakeLists.txt.orig	2019-04-28 14:04:25.000000000 +0200
+++ uriparser-0.9.3/CMakeLists.txt	2019-04-28 16:00:52.723819252 +0200
@@ -292,24 +292,10 @@
 #
 if(URIPARSER_BUILD_DOCS)
     find_package(Doxygen REQUIRED dot doxygen)
-    find_program(QHG_LOCATION
-        NAMES
-            qhelpgenerator
-            qhelpgenerator-qt5  # e.g. CentOS 7
-    )
-    include(FindHTMLHelp)
 
     # Generate Doxyfile
-    if(HTML_HELP_COMPILER)
-        set(GENERATE_HTMLHELP YES)
-    else()
         set(GENERATE_HTMLHELP NO)
-    endif()
-    if(QHG_LOCATION)
-        set(GENERATE_QHP YES)
-    else()
         set(GENERATE_QHP NO)
-    endif()
     configure_file(doc/Doxyfile.in doc/Doxyfile @ONLY)
     configure_file(doc/release.sh.in doc/release.sh @ONLY)
 
