--- uriparser-0.8.5/doc/Makefile.am.orig	2018-02-10 08:58:51.675192042 +0100
+++ uriparser-0.8.5/doc/Makefile.am	2018-02-10 09:00:28.205190940 +0100
@@ -23,9 +23,11 @@
 
 ## Install doc files
 install-data-local:
-	$(MKDIR_P) "$(DESTDIR)$(docdir)/html/search"  ## Didn't work with installdirs-local
+	$(MKDIR_P) "$(DESTDIR)$(docdir)/html"  ## Didn't work with installdirs-local
 	$(INSTALL_DATA) html/*.css html/*.html html/*.js html/*.md5 html/*.png "$(DESTDIR)$(docdir)/html/"
-	$(INSTALL_DATA) html/search/* "$(DESTDIR)$(docdir)/html/search/"
+	if test -d html/search ; then \
+	$(MKDIR_P) "$(DESTDIR)$(docdir)/html/search" ; \
+	$(INSTALL_DATA) html/search/* "$(DESTDIR)$(docdir)/html/search/" ; fi
 	-$(INSTALL_DATA) *.qch "$(DESTDIR)$(docdir)/"
 
 
